app/components/file-editor-column.js

```diff
@@ -18,6 +21,11 @@ export default Ember.Component.extend({
     }
   }),
 
+  lastColumn: computed('col', 'numColumns', function() {
```

Comment:
this CP at first glance appears to indicate, it will provide the last column. But after reading the implementation it seems to actually return true/false. Maybe it should be isLastColumn ?